asset/gulpfile.js

```diff
@@ -40,39 +37,65 @@ var gulp = require('gulp'),
         'yaml'
     ];
 
+// Images
 gulp.task('images', function () {
     return gulp.src('img/*')
         .pipe(gulp.dest('../theme/img/'));
 });
 
+// Icons
+gulp.task('icons', function () {
+    return gulp.src('node_modules/font-awesome/fonts/**.*')
```

Comment:
I'm not sure about it. I can't see point of importing font awesome here. Why not using CDN version?
With bootstrap there is a different story because we modify bootstrap, but FA we are using as it is. Am I right?

Reply:
> Why not using CDN version?

 It is a question of privacy: every request sends (user) data to the CDN.